ci-operator/step-registry/assisted/equinix/assisted-equinix-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: assisted-equinix
+  steps:
+    cluster_profile: packet-assisted
+    allow_best_effort_post_steps: true
+    allow_skip_on_success: true
+    pre:
+      - ref: assisted-equinix-setup
+      - chain: assisted-common-pre
+    test:
+      - ref: assisted-baremetal-conf
+      - ref: assisted-baremetal-test
+    post:
+      - ref: assisted-common-gather
```

Comment:
am I wrong or we're not tearing down the machines?

Reply:
:man_facepalming: